CODEOWNERS

```diff
@@ -1 +1 @@
-*       @canonical/is-charms
+*       @amandahla, @DeeKay3, @nrobinaubertin, @srbouffard
```

Comment:
No commas :)
Also, see we are here you can change it by asking reviews for docs changes to erin except the changelog
Example:
https://github.com/canonical/maubot-operator/blob/main/CODEOWNERS